### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by vasystus**

### CODE REVIEW

**Review by mariaalouisaa — APPROVED** [review]

File: cypress/e2e/spec.cy.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ describe("Parent route", () => {
     cy.visit("/");
     cy.get("h1").contains("Alis Learning");
     cy.get("h1").should("not.contain", "Feedback Form");
-    cy.contains("Go to parents section").click();
+    cy.contains("I'm a parent or teacher").click();
```
Comment (PJSalter) [inline cypress/e2e/spec.cy.js:29]: Good spot on the button texts ✨

File: cypress/e2e/spec.cy.js [hunk 2]
```diff
@@ -87,3 +87,46 @@ describe("window properties", () => {
...
+  it("Check if input name is stored", () => {
+    cy.visit("/learner-onboarding/name");
+    cy.get("#learnerName")
+      .should("be.visible")
+      .type("Potato")
+      .should(() => {
+        expect(localStorage.getItem("learner-name")).to.eq('"Potato"');
+      });
+    cy.visit("/child-landing");
+    cy.get("h2").should("not.contain", "Hi, there");
```
Comment (PJSalter) [inline cypress/e2e/spec.cy.js:102]: I've really enjoyed reading this test, its very clear where by if the childs name is potato  then it shouldn't contain just hi there ✨

File: pages/learner-onboarding/background-selection.js [hunk 3]
```diff
@@ -22,7 +22,7 @@ export default function LearnerBackgroudSelection({
       avatarNameInLocalStorage={avatarNameInLocalStorage}
       background={background}
     >
-      <Grid>
+      <Grid data-testid="background">
```
Comment (PJSalter) [inline pages/learner-onboarding/background-selection.js:25]: Great way to get the test working with cypress 😸 🧪

File: cypress/e2e/spec.cy.js [hunk 4]
```diff
@@ -87,3 +87,46 @@ describe("window properties", () => {
...
+    cy.contains("Hi, Potato!");
+  });
+});
+
+// ✅ --- PASSING ---
+describe("Does the data matches", () => {
```
Comment (PJSalter) [inline cypress/e2e/spec.cy.js:108]: great to see that your testing if the data is there to match each 8 colours, well done 🧪 ✨

File: cypress/e2e/spec.cy.js [hunk 5]
```diff
@@ -87,3 +87,46 @@ describe("window properties", () => {
     cy.title().should("not.contain", "Reading App");
   });
 });
+
+// ✅ --- PASSING ---
+describe("Local storage", () => {
```
Comment (PJSalter) [inline cypress/e2e/spec.cy.js:92]: Awesome work on testing the local storage 🧪 ✨

**Review by PJSalter — APPROVED** [review]

**Review by joe-dev-public — APPROVED** [review]

**Pull request merged**